Source/Core/Gen4/Generators/WildGenerator4.cpp

```diff
@@ -26,9 +26,10 @@
 #include <Core/Util/EncounterSlot.hpp>
 
 WildGenerator4::WildGenerator4(u32 initialAdvances, u32 maxAdvances, u16 tid, u16 sid, u8 genderRatio, Method method,
-                               const StateFilter &filter) :
+                               const StateFilter &filter, bool isPlatinumVersion) :
```

Comment:
I find it difficult that platinum is the only game that is different. Are you sure?

Reply:
Yes i made many tests

Reply:
> I find it difficult that platinum is the only game that is different. Are you sure?

If you check the commit, the only different game isn't Platinum, but Diamond/Pearl. It make sense, cause Diamond and Pearl were released first, so things changed with Pt and HGSS.

Reply:
Can you change the name to just `platinum`